### ISSUE TITLE
Aliased attributes

### ISSUE THREAD

**Issue by mischakolbe** [mischakolbe/node_calculator#64]
Aliased attributes are currently not recognized!

**Comment by mischakolbe** [comment]
This should fix it:

```
def get_mplug_of_mobj(mobj, attr):
    """Get MPlug from the given MObject and attribute combination.

    Args:
        mobj (MObject): MObject of node.
        attr (str): Name of attribute on node.

    Returns:
        MPlug: MPlug of the given node/attr combination.
    """
    node_mfn_dep_node = OpenMaya.MFnDependencyNode(mobj)

    mplug = None

    if node_mfn_dep_node.hasAttribute(attr):
        # findPlug-flags: (attr, wantNetworkedPlug)
        mplug = node_mfn_dep_node.findPlug(attr, False)

    # Check for aliased plugs, if none was found yet.
    if not mplug:
        alias_list = node_mfn_dep_node.getAliasList()
        for alias, attr_name in alias_list:
            if attr == alias:
                mplug = get_mplug_of_mobj(mobj, attr_name)

    return mplug
```

**Comment by mischakolbe** [comment]
Fixed in 2.0.1
